opentelemetry-aws/src/lib.rs

```diff
@@ -86,126 +87,125 @@ pub mod trace {
         _private: (),
     }
 
-    impl XrayPropagator {
-        /// Creates a new `XrayTraceContextPropagator`.
-        pub fn new() -> Self {
-            XrayPropagator::default()
+    pub fn span_context_from_str(value: &str) -> Option<SpanContext> {
```

Comment:
Nit: can we add some documents for those functions now that they are public

Reply:
Add documents